.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: c
-dist: xenial
+dist: bionic
```

Comment:
@plbossart 20.04 was released today :)

Reply:
But not available in Travis yet https://docs.travis-ci.com/user/reference/linux

Reply:
I know, but maybe let's wait until it's really usable... I haven't updated myself just yet for my work devices, only for target ones I can reinstall at will